src/Behat/Behat/Transformation/Context/Annotation/TransformationAnnotationReader.php

```diff
@@ -64,18 +64,13 @@ public function readCallee($contextClass, ReflectionMethod $method, $docLine, $d
      */
     private function simpleTransformations()
     {
-        $transformations = array();
-        $transformations[] = 'Behat\Behat\Transformation\Transformation\RowBasedTableTransformation';
-        $transformations[] = 'Behat\Behat\Transformation\Transformation\ColumnBasedTableTransformation';
-        $transformations[] = 'Behat\Behat\Transformation\Transformation\TableRowTransformation';
-
-        if (PHP_VERSION_ID >= 70000) {
-            $transformations[] = 'Behat\Behat\Transformation\Transformation\TokenNameAndReturnTypeTransformation';
-            $transformations[] = 'Behat\Behat\Transformation\Transformation\ReturnTypeTransformation';
-        }
-
-        $transformations[] = 'Behat\Behat\Transformation\Transformation\TokenNameTransformation';
-
-        return $transformations;
+        return array(
```

Comment:
you may as well use `[]` for the array

Reply:
I've thought about that, but the risk of doing that is that you'll end up with a mix of both styles throughout the entire codebase. So I thought it would be better to at least keep consistent now with how it was and than later replace all of it in one swoop. Probably using something like Rector.